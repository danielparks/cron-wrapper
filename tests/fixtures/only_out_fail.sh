#!/bin/bash

echo out
exit 1
