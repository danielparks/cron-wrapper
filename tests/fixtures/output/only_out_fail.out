Command: ["tests/fixtures/only_out_fail.sh"]
Start: 2023-07-26T13:32:23.254904000-07:00

0.003 out out
0.003 exit 1
